GitUI/Theming/ThemeRepository.cs

```diff
@@ -8,16 +8,19 @@
 
 namespace GitUI.Theming
 {
-    public class ThemeRepository
+    public class ThemeRepository : ICssUrlResolver
     {
         private const string Subdirectory = "Themes";
         private const string Extension = ".css";
+        private const string CssVariableUserThemesDirectory = "$user-defined/";
```

Comment:
For scripts we use `{variable}` notation, is it possible to use the same, e.g. `{UserAppData}/`?

Reply:
ok